src/Protocol/ActivityPub/Receiver.php

```diff
@@ -98,7 +97,7 @@ public static function processInbox($body, $header, int $uid)
 
 		$actor = JsonLD::fetchElement($ldactivity, 'as:actor', '@id');
```

Comment:
`$actor` is used a few times, might as well do the null-coalescing earlier.

```suggestion
		$actor = JsonLD::fetchElement($ldactivity, 'as:actor', '@id') ?? '';
```

Reply:
Okay, that proposed change makes sense!